01-js-intro-and-jquery/javascript.js

```diff
@@ -0,0 +1,39 @@
+$(document).ready(function() {
+	
+	$("#txtName").focus();
+
+
+	$("#btnSearch").click(function (){
+
+		$('li').fadeOut("slow", function(){$(this).remove();});
+		var search = document.getElementById("txtSearch").value;
```

Comment:
Based on hoisting all variables should be defined on the upper top of the function